README.md

```diff
@@ -58,7 +103,7 @@ This is simply by far the most used framework used in developing front-end appli
 
 # Coding Challenges
 
-This is a collection of coding challenges to get you warmed up. The end goal is to eventually build an NFT! The coding challenges will help you get a bit of an understanding of things. Remember that sometimes there are multiple ways to get to a solution, and do not feel ashamed to look things up online or ask questions if you are lost.
+This is a collection of coding challenges to get you warmed up. The end goal is to eventually build an NFT! The coding challenges will help you get a bit of an understanding of things. Remember that sometimes there are multiple ways to get to a solution, and do not feel ashamed to look things up online or ask questions if you are lost. The hackathon is not a complete solidity tutorial - we will show you things, but if you want a deeper understanding, there are tons of good guides and videos out there for you to review later on or during the hackathon if there is time.
```

Comment:
I'd call it: "The workshop" 
And then say, "The workshop is a collection of coding challenges to get you warmed up with solidity development in General. It is not a complete solidity tutorial as that is too much to learn in one day but it is a good primer to grasp the overall concept. The end goal of this workshop is to enable you to build your own NFT!"
Followed by everything else you're saying here
And then the re